free-1.md

```diff
@@ -57,12 +136,48 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think around 50 percent of the time would be spent on desigining the language.
+I feel like it will be fairly simple to define terms of the lanugage, especially
+because coding languages for visual art already exist and I would be able to model
+aspects of my language after those. However, I think I'll need to make important 
+design decisions on how the the code can be organized from the user end (i.e. 
+creating classes or separate files). I also think I'll need to decide how simple
+the language will be, either by limiting the vocabulary or by limiting the
+parameters for different objects.
+
+The other half of the time will be spent figuring out how to parse and execute 
+the code, figuring out how the shapes and animations will be represented on the 
```

Comment:
I think drawing the shapes and doing the actual animation could end up being quite tricky, especially if you want something lightweight, accessible, and portable. See comment above w.r.t HTML/CSS/SVG + webkit.